app/api/domains/osu.py

```diff
@@ -1301,12 +1301,17 @@ async def get_leaderboard_scores(
     if leaderboard_type == LeaderboardType.Mods:
         query.append("AND s.mods = :mods")
         params["mods"] = mods
+        query.append("AND s.status IN (1, 2)")  # 1 = submitted, 2 = best
     elif leaderboard_type == LeaderboardType.Friends:
         query.append("AND s.userid IN :friends")
         params["friends"] = player.friends | {player.id}
+        query.append("AND s.status = 2")
     elif leaderboard_type == LeaderboardType.Country:
         query.append("AND u.country = :country")
         params["country"] = player.geoloc["country"]["acronym"]
+        query.append("AND s.status = 2")
+    else:
+        query.append("AND s.status = 2")
```

Comment:
Why not use one `if` instead of the 3 same lines?

Like this - 
```py
if leaderboard_type != LeaderboardType.Mods:
    query.append("AND s.status = 2")
```

Reply:
sure